etc/kayobe/environments/ci-aio/globals.yml

```diff
@@ -55,11 +55,6 @@ os_release: >-
           {{ (lookup('pipe', '. /etc/os-release && echo $VERSION_CODENAME') | trim) if os_distribution == 'ubuntu' else
              (lookup('pipe', '. /etc/os-release && echo $VERSION_ID') | trim | split('.') | first) if os_distribution == 'rocky' }}
 
-###############################################################################
-
-# Avoid a reboot.
-selinux_state: disabled
```

Comment:
Why remove this?

Reply:
I was always unsure about this but I removed it because I saw it was on the list [here](https://docs.openstack.org/releasenotes/kayobe/2023.1.html#:~:text=*%20%60%60change_selinux%60%60)  so I assumed it wouldn't work anymore

Reply:
Ah, I see. Your link points to change_selinux which is no longer relevant, but selinux_disabled is now a Kayobe variable so should be moved to the Kayobe namespace.

Reply:
I can't find `selinux_disabled` referenced anywhere in kayobe. Did you mean `selinux_state`?

Reply:
But don't we want to keep this?

Reply:
Re-added